CITATION.cff

```diff
@@ -0,0 +1,11 @@
+cff-version: 1.2.0
+title: OJS Plum Analytics plugin (Unofficial fork)
+message: "If you use this software, please cite it as below."
+authors:
+  - family-names: Al-Slfi
```

Comment:
Hi @saddamalsalfi, while I can't speak for the dev team on the plugin, I think this should probably not be included in this PR as it only applies to your fork and not the main repository here.